setup.py

```diff
@@ -67,7 +65,7 @@ def run(self):
 ]
 
 package_name = os.environ.get("GALAXY_NG_ALTERNATE_NAME", "galaxy-ng")
-version = os.environ.get("ALTERNATE_VERSION", __version__)
+version = "4.3.0.dev"
```

Comment:
Is there a way to read this version from `setup.cgf` ? parse it and take the `bumpversion.current_version` value?
and also keep the lookup for `ALTERNATE_VERSION` ?

Reply:
> Is there a way to read this version from setup.cgf ? 

I think yes.

> parse it and take the bumpversion.current_version value?

AFAIK bumpversion simply parses file with regexp and replaces version in it. So it should be trivial to move metadata (along with version) from `setup.py` to `setup.cfg`